website/docs/cookbooks/scala-versions.md

```diff
@@ -0,0 +1,71 @@
+# Picking Scala version with scala-cli
+
+Scala cli by default runs latest stable Scala version.
+
+Here is an unversal piece of code that detect Scala version in runtime
+
+```scala
+object ScalaVersion extends App {
```

Comment:
I run locally this example, it throws me the following error:
```
Compiling project (Scala 3.0.2, JVM)
[error] ./ScalaVersion.scala:7:38: Not found: versions
[error]                                      ^^^^^^^^^
Error compiling project (Scala 3.0.2, JVM)
Compilation failed
```

Reply:
Yes, that is why I added automated validation as a followup :)